index.html

```diff
@@ -19,34 +19,78 @@
         <li class="logo-link"><a href="./index.html">QUIZ NAME</a></li>
       </ul>
       <ul class="navigation-list">
-        <li><a href="/pages/actors.html">Actors</a></li>
-        <li><a href="/pages/frames.html">Movie Frames</a></li>
-        <li><a href="/pages/sounds.html">Soundtracks</a></li>
+        <li>
+          <button href="/pages/actors.html" class="js-quiz-target actors-click" data-target="actors">Actors</button>
```

Comment:
Jak to już nie są linki to można usunąć href z buttonów. A także klasy actors-click itd., które ostatecznie nie są używane